### ISSUE TITLE
Uncaught TypeError: decorator is not a function

### ISSUE THREAD

**Issue by gautier-lefebvre** [andreypopp/autobind-decorator#56]
With the latest release 1.4.4, we get an error when using the autobind on a method, and building with webpack and `babel-plugin-transform-decorators-legacy`:

```js
class Suggestion extends PureComponent {
    @autobind
    _onClick() {
        return this.props.onClick(this.props.suggestion);
    }
    ...
}
```

inside _applyDecoratedDescriptor (from webpack plugin) ->

```
Uncaught TypeError: decorator is not a function
```

**Comment by longlho** [comment]
I believe babel 6 (and 7) no longer retrofit `module.exports`, so if ur using autobind via `require` u'd have to do `require('autobind-decorator').default`

**Comment by gautier-lefebvre** [comment]
Yeah you're probably right, but that's a breaking change, since previous versions were working fine with babel 6. That would be great if you released these changes as a new major version.

**Comment by longlho** [comment]
@stevemao ^^

**Comment by stevemao** [comment]
unpublished `1.4.4` and published `2.0.0`.
